package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "grafana-iot-twinmaker-app",
-  "version": "1.2.1",
+  "version": "1.2.2",
```

Comment:
do we need to bump version in this PR? we should at least bump its minor version to `1.3.0`

Reply:
We've already updated the version a recent commit on main: https://github.com/grafana/grafana-iot-twinmaker-app/blob/main/package.json#L3